### PR TITLE
Full namespace changes

### PR DESCRIPTION
**Pull request by cartoon-face**
This pull request follows on from https://github.com/ConcealNetwork/conceal-core/pull/239. @AxVultis mentioned wallets could no longer create a mnemonic seed, this now fixes that.

### Wallet Test
```
Wallet Address: ccx7Dc5HY8pboTVyHzxsPoPvnWe2PRRcLNPhKLx5a8kaLbnJzJoGZ5m76RGDCyXs7UdZNjfXgkgsU5NuyEN4G5MF9dr7LWNNro
Private spend key: 1773787e1a048bbb1844a9fb5fbe76e102046d2a4569db2005e8d5e940b44b01
Private view key: 8562d01ef567a3b9144a7d96a012b500c1701427ad5848b3ea0f71ca3e789502
Mnemonic Seed: dexterity ruffled earth asleep code umbrella border axes artistic joining cunning answers rebel vane bakery fibula snout utmost reorder napkin ladder gopher newt nineteen artistic
```

### Full Changes

`CryptoNote` = `cn`
`Crypto` = `crypto`
`Common` = `common`
`Logging` = `logging`
`TransactionTypes` = `transaction_types`
`Tools` = `tools`
`Base58` = `base_58`
`Base64` = `base_64`
`Console` = `console`
`System` = `platform_system` (namespace `system` causes conflictions)
`Miner` = `miner`
`Language` = `seed_language` (namespace `language` causes runtime conflictions because of declarations declared as "language")
`ElectrumWords` = `electrum_words`
`PaymentService` = `payment_service`

### CODE REVIEW

File: src/Miner/Miner.h [hunk 1]
```diff
@@ -18,39 +18,39 @@
...
-namespace CryptoNote {
+namespace cn {
 
 struct BlockMiningParameters {
   Block blockTemplate;
   difficulty_type difficulty;
 };
 
-class Miner {
+class miner {
```
Comment (AxVultis) [inline src/Miner/Miner.h:28]: The class should remain in CamelCase here
```suggestion
class Miner {
```

**Review by AxVultis — COMMENTED** [review]
Great work, I made one suggestion and everything will be good

**Comment by cartoon-face** [comment]
Everything looks good now, you are definitely right about CamelCasing the class. I accidentally got the miner class and namespace both mixed up but this is solved now. This should be ready for you now.

**Comment by krypt0x** [comment]
Great job! We'll review and merge it ASAP. Thank you very much.

File: src/CryptoNoteCore/Miner.h [hunk 2]
```diff
@@ -22,10 +22,10 @@
 
 #include "Serialization/ISerializer.h"
 
-namespace CryptoNote {
+namespace cn {
   class miner {
```
Comment (AxVultis) [inline src/CryptoNoteCore/Miner.h:26]:
```suggestion
  class Miner {
```

**Review by AxVultis — COMMENTED** [review]
It seems the class is still not in CamelCase

**Comment by cartoon-face** [comment]
I reverted the class name back to its original state as seen here in the master branch: https://github.com/ConcealNetwork/conceal-core/blob/4a75359e2b62d71ce5ffec0d10c05f55c045559d/src/CryptoNoteCore/Miner.h#L26
I can change the whole class to CamelCase if you’d like with the changes already made

**Review by AxVultis — APPROVED** [review]
I mixed up with another file, sorry.
It is all good for me.
Thanks for your work.

**Comment by cartoon-face** [comment]
To be honest, I think I did upon first making these changes because we have CryptoNoteCore/Miner plus Miner/Miner.
Glad I can help out around here.

**Review by krypt0x — APPROVED** [review]
I think everything has been covered. Let's merge it into `namespaces` branch and test it. Thank you.

**Pull request merged**